packages/account/src/Components/text-container/__test__/text-container.spec.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { TextContainer } from '../text-container';
+
+describe('<TextContainer/>', () => {
+    it('should contain "Lorem Ipsum" in the document', () => {
+        render(
+            <TextContainer children='Lorem Ipsum'/>
+        );
+
+        expect(screen.getByText(/Lorem Ipsum/i)).toBeInTheDocument();
```

Comment:
here you can also test text container is rendered
expect(screen.getByTestId(/dt_text_container/i)).toBeInTheDocument();